change/react-native-windows-2020-05-06-14-46-56-Issue4792.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
This isn't the change type you want to be using. "none" is used to indicate no changes were made to the package. This means it doesn't trigger a publish or get added to the package changelog.

You should usually choose "prerelease" when generating the change instead. For this change you can just manually change "none" to "prerelease" in this file.

Reply:
I thought "publishing" refers to the product binary. As your concern indicates, that's not true. Which bits _are_ we actually publishing?